libs/sdk-backend-tiger/src/backend/workspaces/index.ts

```diff
@@ -69,66 +71,12 @@ class TigerWorkspaceQuery implements IWorkspacesQuery {
             next: () => Promise.resolve(emptyResult),
         };
 
-        /*
-         * Tiger has no service to obtain list of workspaces yet. Thus hardcoding all available workspaces
-         * it may support.
-         *
-         * List taken from NAS code: sqlexecutor/databaseaccess/DataSourceService.kt
-         */
-        const workspaces: IAnalyticalWorkspace[] = [
-            {
-                title: "TPC-H - Postgres",
-                id: "tpch",
-                description: "",
-                isDemo: true,
-            },
-            {
-                title: "GoodSales - Postgres",
-                id: "goodsales",
-                description: "",
-                isDemo: true,
-            },
-            {
-                title: "UFO - Postgres",
-                id: "ufo",
-                description: "",
-                isDemo: true,
-            },
-            {
-                title: "COVID-19 - Postgres",
-                id: "covid",
-                description: "",
-                isDemo: true,
-            },
-            {
-                title: "TPC-H - Redshift",
-                id: "tpch_rs",
-                description: "",
-                isDemo: true,
-            },
-            {
-                title: "GoodSales - Redshift",
-                id: "goodsales_rs",
-                description: "",
-                isDemo: true,
-            },
-            {
-                title: "UFO - Redshift",
-                id: "ufo_rs",
-                description: "",
-                isDemo: true,
-            },
-            {
-                title: "COVID-19 - Redshift",
-                id: "covid_rs",
-                description: "",
-                isDemo: true,
-            },
-        ].map((descriptor) => new TigerWorkspace(this.authCall, descriptor.id, this.dateFormatter));
-
         return {
             search,
-            items: workspaces,
+            items: workspaces.map((workspace: WorkspaceObjectModel.IWorkspace) => {
```

Comment:
We should also implement the `next` property, otherwise there is no way to access pages other than the first. Please see bear version of this for inspiration.